src/server/index.js

```diff
@@ -11,6 +12,7 @@ appModulePath.addPath(path.join(__dirname, '..', '..', 'src'))
 const routes = require('./routes')
 const app = express()
 
+app.use(cors())
```

Comment:
this should work on prod, eh?

Reply:
Removed it! Added this earlier but not needed now